Cargo.toml

```diff
@@ -27,7 +27,7 @@ paw = ["structopt-derive/paw"]
 travis-ci = { repository = "TeXitoi/structopt" }
 
 [dependencies]
-clap = { version = "2.33", default-features = false }
+clap = { git = "https://github.com/davidMcneil/clap.git", branch = "v2-master", default-features = false }
```

Comment:
Sorry for being blunt, but this is something we will definitely not accept. Only proper versions are our dependencies.